io.catenax.shared.business_partner_certificate/1.0.0/BusinessPartnerCertificate.ttl

```diff
@@ -0,0 +1,171 @@
+#######################################################################
+# Copyright (c) 2023-2024 Contributors to the Eclipse Foundation

```

Comment:
only 2024 is required.

Reply:
you did not add your teams name in copyright headers?

Reply:
Added team details asn adapted year as 2024